public/styles/style.css

```diff
@@ -1,3 +1,106 @@
+html {
```

Comment:
This is generated code?
if so
1. it makes qa harder
2. it grows the size of .git file
3. sometimes can result in nasty merge conflicts.

Atm there is no description of build step in readme..